integration-tests/pino.spec.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable comma-dangle */
+'use strict'
+
+const { FakeAgent, spawnProc, createSandbox, curl } = require('./helpers')
+const path = require('path')
+const { assert } = require('chai')
+const { once } = require('events')
+
+describe('pino test', () => {
+  let agent
+  let proc
+  let sandbox
+  let cwd
+  let startupTestFile
+
+  before(async () => {
+    sandbox = await createSandbox(['pino'])
+    cwd = sandbox.folder
+    startupTestFile = path.join(cwd, 'pino/index.js')
+  })
+
+  after(async () => {
+    await sandbox.remove()
+  })
+
+  context('Log injection', () => {
+    beforeEach(async () => {
+      agent = await new FakeAgent().start()
+    })
+
+    afterEach(async () => {
+      proc.kill()
+      await agent.stop()
+    })
+
+    it('Log injection enabled', async () => {
+      proc = await spawnProc(startupTestFile, {
+        cwd,
+        env: {
+          AGENT_PORT: agent.port,
+          lOG_INJECTION: true,
+        },
+        stdio: 'pipe',
+      })
+      const [data] = await Promise.all([once(proc.stdout, 'data'), curl(proc)])
+      const stdoutData = JSON.parse(data.toString())
+      assert.containsAllKeys(stdoutData, ['dd'])
+      assert.containsAllKeys(stdoutData.dd, ['trace_id', 'span_id'])
+      assert(stdoutData['dd']['trace_id'] === stdoutData['custom']['trace_id'])
```

Comment:
For this and the next line, you should use https://www.chaijs.com/api/assert/#method_strictequal

e.g.

```js
assert.strictEqual(stdoutData.dd.trace_id, stdoutData.custom.trace_id)
```